src/Microsoft.FeatureManagement/FeatureFilters/TimeWindowFilter.cs

```diff
@@ -37,7 +38,7 @@ public object BindParameters(IConfiguration filterParameters)
         }
 
         /// <summary>
-        /// Evaluates whether a feature is enabled based on a configurable time window.
+        /// Evaluates whether a feature is enabled based on a configurable fixed time window or recurring time windows.
```

Comment:
I don't think we need to update every summary to mention fixed or recurring. Ideally that's a one time thing and we figure out how to generalize elsewhere.

Reply:
How about "Evaluates whether a feature is enabled based on the <see cref="TimeWindowFilterSettings"/> specifed in the configuration."?